samples/csx/HttpSyncStart/run.csx

```diff
@@ -15,8 +20,33 @@ public static async Task<HttpResponseMessage> Run(
     string instanceId = await starter.StartNewAsync(functionName, eventData);
     
     log.Info($"Started orchestration with ID = '{instanceId}'.");
-    
-    return await starter.WaitForCompletionOrCreateCheckStatusResponseAsync(
-        req,
-        instanceId);
+
+    //Get query string parameters and add them to a dictionary
+    var queryPairs = req.RequestUri.Query.Replace("?", string.Empty)
+        .Split(new[] {"&"}, StringSplitOptions.None);
+    var parameters = queryPairs.Select(
+        item => item.Split(new[] {"="}, StringSplitOptions.None))
+        .Where(queryPairElementsArray => queryPairElementsArray.Length > 1).ToDictionary(
+        queryPairElementsArray => queryPairElementsArray[0],
+        queryPairElementsArray => queryPairElementsArray[1]);
```

Comment:
I don't feel good about the DIY method of query string parameter parsing. I worry that this complexity distracts from the point of this sample, which is how to start an orchestrator function synchronously. The built-in Azure Functions HTTP trigger samples use `GetQueryNameValuePairs()`, and I'd prefer we do the same here. I think it's okay to diverge the implementation between csx and precompiled if necessary.

Reply:
Great!

Reply:
As we agreed I will set the values explicitly in the source code before the next updates.